merge_sort.py

```diff
@@ -0,0 +1,55 @@
+def merge_sort(unord_list):
+    return merge_helper(
+        unord_list[:(len(unord_list)/2)],
+        unord_list[(len(unord_list)/2):]
+        )
+
+
+def merge_helper(left, right):
+    """A sorting algorithm that splits an unorderd list in half,
+    repeatedly, until it is reduced to individual chunks, then
+    sorts the chunks."""
+    if len(right) == 1 and len(left) == 1:
+        return [min(left[0], right[0]), max(left[0], right[0])]
+
+    if len(right) == 0 or len(left) == 0:
+        return right
+
+    left = merge_helper(left[:(len(left)/2)], left[(len(left)/2):])
+    right = merge_helper(right[:(len(right)/2)], right[(len(right)/2):])
+    l = []
+    for var in right:
+        while len(left) > 0 and left[0] < var:
+            l.append(left.pop(0))
+        l.append(var)
```

Comment:
This is quite tidy!

I really don't like the `list.pop(0)` thing though.  Wish there were a way around it.